### PR TITLE
Create configs.csv file

### PR DESCRIPTION
**Pull request by 3dm1**

### CODE REVIEW

File: cameras/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Fields
+| Field | Description | Required |
+| ----- | ----------- | -------- |
+| company	| The name of the camera manufacturer, e.g. "Sharx" or "Amcrest" | required |
+| type | `camera` for standalone IP cameras, or `nvr` for Network Video Recorders | required |
+model	| The model name | required |
+oui_regex	| The regular expression that matches the OUI of the MAC Address. It's OK to assume uppercase only with colon separators like `(00:80:F0)` | required |
+rtsp_url	| The full url that reaches the H.264 RTSP video stream | required |
+stream_name	| A `;` separated list of user-facing description of each stream, e.g `high-res 1080p; low-res 640p`. Defaults to `Stream #{{stream}}` | optional |
```
Comment (jcmaslan) [inline cameras/README.md]: `stream_names` and `channel_names` just for symmetry with `streams` and `channels`

File: configs/configs.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+maker,command_id,url,method,auth_type,regex_success,regex_firmware,regex_model
+Sharx Security,enable_wifi,http://{{ip_address}}/form/setWlan/?ENABLE=1&SSID={{wifi_ssid}}&MODE=INFRA&SECURITY=WPA2-PSK&ENCRYPTION=AES&AUTHTYPE=AUTO&KEYTYPE=HEX&WEPIDX=1&WEPKEY=&WPAKEY={{wifi_password}},GET,basic,^200 Success,^v5,.*SCNC3[89]0[45].*
```
Comment (jcmaslan) [inline configs/configs.csv]: `regex_success` should be:
`^000 Success.*`

File: configs/README.MD [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Fields
+
+| Field | Description | Required |
+| ----- | ----------- | -------- |
+| maker	| The name of the camera manufacturer, e.g. "Sharx" or "Amcrest" | required |
+| command_id	| An unique identifier string for the command, e.g. "enable_wifi" | required |
+| url	| The full url used to execute the camera configuration | required |
+| method | The method used to access `url`, e.g. "GET" | required |
+| regex_model	| A regex representing all the models that support the comamnd, e.g "SCNC3[89]0[45]" | required |
+| regex_firmware	| A regex representing all the firmware versions that support the comamnd, e.g "^v4" | required |
```
Comment (jcmaslan) [inline configs/README.MD:15]: `auth_type` as optional.
The camera's server _should_ reply with `WWW-Authenticate: Digest` to tell us that we should try `Authorization: Digest` , but I guess we can include this optional value for now. 
We may need to re-think this and just automatically try `Authorizaiton: Digest ...` whenever `Authorization: Basic ...` fails. So we may eliminate this field.

Comment (3dm1) [inline configs/README.MD:15]: Shouldn't this be in the configs.csv documentation?

Comment (jcmaslan) [inline configs/README.MD:15]: Isn't this README.MD the configs doc? This is the place to document the configs.csv right?

Comment (3dm1) [inline configs/README.MD:15]: Ignore me, you are right.

**Pull request merged**